requirements.in

```diff
@@ -21,15 +21,14 @@ pyexcel-ods3==0.6.1
 pytz==2022.6
 # Should be pinned until a new gunicorn release greater than 20.1.0 comes out. (Due to eventlet v0.33 compatibility issues)
 git+https://github.com/benoitc/gunicorn.git@1299ea9e967a61ae2edebe191082fd169b864c64#egg=gunicorn[eventlet]==20.1.0
-notifications-python-client==6.4.1
+notifications-python-client==8.0.0
```

Comment:
are we meant to be updating this?

Reply:
The notify client currently isn't used anywhere. But I'd like to be on the latest version ahead of the email/sms integration ticket.